google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -373,10 +373,14 @@ def visit_TEXT(self, type_, **kw):
     def visit_ARRAY(self, type_, **kw):
         return "ARRAY<{}>".format(self.process(type_.item_type, **kw))
 
-    def visit_BINARY(self, type_, **kw):
+    def visit_BINARY(self, type_, **kw):  # pragma: no cover
+        """
+        The BINARY type is superseded by large_binary in
+        newer versions of SQLAlchemy (>1.4).
+        """
         return "BYTES({})".format(type_.length or "MAX")
 
-    def visit_large_binary(self, type_, **kw):
+    def visit_large_binary(self, type_, **kw):  # pragma: no cover
```

Comment:
Here we have two data types. Binary is older and it's used before SQLAlchemy 1.4 version. Large binary is a newer type, which is used by new SQLAlchemy versions. As we're supporting both version ranges, we'll always get in a situation when one type was used, but not another one. As this will influence coverage percent, I'd skip them both in report.